### ISSUE TITLE
传奇你好，使用您的与训练模型，然后用自己的训练数据集训练行人检测模型，8000样本，迭代了80000次，detection_eval = 0.581653，loss  loss = 2.67537，您能简单的指导下问题可能处在什么地方了吗？

### ISSUE THREAD

**Issue by Suning-liuhaiyang** [chuanqi305/MobileNet-SSD#42]

**Comment by chuanqi305** [comment]
1、batch数调一下试试，有时较大的batch会更有效
2、学习速率一开始不能太大，甚至固定基础网络的权重只训练最后一层
3、建议将voc的person类的权重提取出来作为你的基础网络，这样一开始的loss就不会很大，在这个基础上finetune会好一些

**Comment by dlyldxwl** [comment]
你好！ 我在提取预训练网络某几类的权重的时候，比如说conv16_2_mbox_conf层, weight.shape为[126,512,1,1]，我现在提取4类物体，需要的weight.shape就是[24,512,1,1]。在原始的weight的21*6=126中，我是每隔6个取一组数据吗，是不是同一类的weight是在一起的，然后是按照voc的labelmap里面的类别的顺序来确定的吗？如果是的话，作者可以提供一下当时训练预训练模型的labelmap吗？
深表感谢！

**Comment by dlyldxwl** [comment]
如果是检测bicycle的话，对应于voc_labelmap是第三类，那就是提取weight的126维中13-18组数据吗？

**Comment by dlyldxwl** [comment]
I get，weight不是按照6*21排列的，是按照21*6排列的。

**Comment by dlyldxwl** [comment]
I get，weight不是按照6×21排列的，是按照21×6排列的。

**Comment by plantree** [comment]
@liu-hai-yang 请问怎么构造自己的训练集？

**Comment by Danbinabo** [comment]
和我训练人脸检测的效果差不多，nmp有点低呀
